src/basho_bench_worker_sup.erl

```diff
@@ -58,6 +62,21 @@ stop_child(Id) ->
 active_workers() ->
     [X || X <- workers(), X =/= undefined].
 
+worker_count() ->
+    case whereis(?MODULE) of
+        undefined ->
+            case erlang:get(last_worker_count) of
```

Comment:
For crash recovery?

Reply:
This is a hack around a bit of awkward process bootstrapping; in particular, the `basho_bench_stats` process starts prior to the `basho_bench_worker_sup` and also potentially post the termination of the worker sup pid when we're still doing stats collection (which is the purpose of the `last_worker_count` value). We might be able to rework this at some point, but everything else here was fairly simple so I went with this approach.